Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -1206,8 +1040,7 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
         userSettingsDisplayNameIndex = 1;
         userSettingsChangePasswordIndex = 2;
         userSettingsEmailStartIndex = 3;
-        userSettingsNewEmailIndex = userSettingsEmailStartIndex + account.linkedEmails.count;
-        userSettingsPhoneStartIndex = userSettingsNewEmailIndex + 1;
+        userSettingsPhoneStartIndex = userSettingsEmailStartIndex + 1;
```

Comment:
I would keep here the possibility to display several linked emails (even if only one email is expected for the moment)